ddht/v5_1/alexandria/partials/proof.py

```diff
@@ -164,6 +311,71 @@ def get_tree(self) -> ProofTree:
             self._tree_cache = ProofTree(root_node)
         return self._tree_cache
 
+    def serialize(self) -> bytes:
```

Comment:
nit: feel like I've been spoiled with the great recipe walkthroughs; wouldn't mind another here.